spec/support/features/funding_details_steps.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Features
+  module FundingDetailsSteps
+    def and_the_funding_details_is_complete
+      review_draft_page.funding_section.link.click
+      edit_funding_page.training_initiative_radios.choose("Now teach")
```

Comment:
Is "Now teach" a training initiative available for all routes? Maybe change this to
```suggestion
      edit_funding_page.training_initiative_radios.first.choose
```
to make it future proof for new routes

Reply:
ah it is available for all routes 🤷